src/main/java/lotto/StringAddCalculator.java

```diff
@@ -0,0 +1,43 @@
+package lotto;
+
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+	private static String DEFAULT_DELIMITER = ",|:";
+
+	public static int splitAndSum(String inputText) {
+		inputText = Optional.ofNullable(inputText).map(String::valueOf).orElse("");
```

Comment:
변수를 새로 할당하면 되지 않나요?
외부 객체나 변수의 값에 영향을 주면 안됩니다

```suggestion
		String text = Optional.ofNullable(inputText)
			.orElse("");
```